Source/WebInspectorUI/UserInterface/Controllers/TimelineManager.js

```diff
@@ -1215,6 +1217,17 @@ WI.TimelineManager = class TimelineManager extends WI.Object
         this._addRecord(new WI.ResourceTimelineRecord(mainResource));
     }
 
+    _handleMessageAdded(event)
+    {
+        if (!this._enabled)
+            return;
```

Comment:
Should we also make sure the screenshot timeline itself is enabled?

Reply:
hmm probably?  i feel like this is already handled somewhere inside `_addRecord`.  will investigate

Reply:
this should already be handled by `WI.ScreenshotsInstrument.supported()`, as once that's `true` we will always add a `WI.Timeline` for it (even though we may not show it) in `WI.TimelineRecording`

we also probably want to do this regardless so that if the developer decides to enable the Screenshots timeline later on, they can see all the images that were captured